inc/datahandlers/pm.php

```diff
@@ -293,20 +293,15 @@ function verify_recipient()
 			// - sender is an administrator
 			if(($this->admin_override != true && $sender_permissions['cancp'] != 1) && $sender_permissions['canoverridepm'] != 1)
 			{
-				$ignorelist = explode(",", $user['ignorelist']);
-				if(!empty($ignorelist) && in_array($pm['fromid'], $ignorelist))
+				if(!empty($user['ignorelist']) && my_strpos(','.$user['ignorelist'].',', ','.$pm['fromid'].',') !== false)
```

Comment:
`strpos()` is sufficient here, there are no Unicode characters in `$user['ignorelist']`/`$user['buddylist']`